### DISCUSSION TITLE
Create a topic for Tasks...

### DISCUSSION THREAD

**Discussion by BCFCODE** [Sepehrsalimian/WebStarter#3] [Ideas]
#Tasks @Sepehrsalimian  Start a new discussion in the Announcements category and title it 'Tasks.'"

**Comment by Sepehrsalimian** [comment]
Done!
@BCFCODE

**Reply by BCFCODE** [reply]
![image](https://github.com/user-attachments/assets/14ae96ce-518b-4fba-808f-082770bf03bc)
Good, but just a heads-up: I closed this because I had a change of heart regarding the task, topic, or issue. @BCFCODE

**Reply by Sepehrsalimian** [reply]
I got it. Thanks
@BCFCODE
